[2025-07-25T11:10:50.246+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-25T11:10:50.271+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt_to_dwh.extract_task.load_db_to_parquet scheduled__2025-07-10T00:00:00+00:00 [queued]>
[2025-07-25T11:10:50.285+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt_to_dwh.extract_task.load_db_to_parquet scheduled__2025-07-10T00:00:00+00:00 [queued]>
[2025-07-25T11:10:50.287+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-07-25T11:10:50.309+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): extract_task.load_db_to_parquet> on 2025-07-10 00:00:00+00:00
[2025-07-25T11:10:50.324+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'elt_to_dwh', 'extract_task.load_db_to_parquet', 'scheduled__2025-07-10T00:00:00+00:00', '--job-id', '73', '--raw', '--subdir', 'DAGS_FOLDER/elt_to_dwh.py', '--cfg-path', '/tmp/tmp7nxyujo0']
[2025-07-25T11:10:50.327+0000] {standard_task_runner.py:105} INFO - Job 73: Subtask extract_task.load_db_to_parquet
[2025-07-25T11:10:50.330+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2733) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-25T11:10:50.331+0000] {standard_task_runner.py:72} INFO - Started process 2743 to run task
[2025-07-25T11:10:50.392+0000] {task_command.py:467} INFO - Running <TaskInstance: elt_to_dwh.extract_task.load_db_to_parquet scheduled__2025-07-10T00:00:00+00:00 [running]> on host 74cfddf97016
[2025-07-25T11:10:50.474+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='elt_to_dwh' AIRFLOW_CTX_TASK_ID='extract_task.load_db_to_parquet' AIRFLOW_CTX_EXECUTION_DATE='2025-07-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-10T00:00:00+00:00'
[2025-07-25T11:10:50.476+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-07-25T11:10:50.521+0000] {taskinstance.py:3313} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 768, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 734, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/scripts/elt_to_dwh/load/load_db_to_parquet.py", line 21, in load_db_to_parquet
    df = pl.read_database(query=query, connection=conn)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/polars/io/database/functions.py", line 249, in read_database
    return cx.execute(
           ^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/polars/io/database/_executor.py", line 501, in execute
    result = cursor_execute(query, **options)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.SyntaxError: syntax error at or near "WHERE"
LINE 27: WHERE
         ^

[2025-07-25T11:10:50.535+0000] {taskinstance.py:1226} INFO - Marking task as FAILED. dag_id=elt_to_dwh, task_id=extract_task.load_db_to_parquet, run_id=scheduled__2025-07-10T00:00:00+00:00, execution_date=20250710T000000, start_date=20250725T111050, end_date=20250725T111050
[2025-07-25T11:10:50.560+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-07-25T11:10:50.561+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 73 for task extract_task.load_db_to_parquet (syntax error at or near "WHERE"
LINE 27: WHERE
         ^
; 2743)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3006, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 274, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3161, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3185, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 768, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 734, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/scripts/elt_to_dwh/load/load_db_to_parquet.py", line 21, in load_db_to_parquet
    df = pl.read_database(query=query, connection=conn)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/polars/io/database/functions.py", line 249, in read_database
    return cx.execute(
           ^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/polars/io/database/_executor.py", line 501, in execute
    result = cursor_execute(query, **options)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.SyntaxError: syntax error at or near "WHERE"
LINE 27: WHERE
         ^

[2025-07-25T11:10:50.589+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-07-25T11:10:50.607+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-25T11:10:50.614+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-07-25T11:50:20.458+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-25T11:50:20.471+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt_to_dwh.extract_task.load_db_to_parquet scheduled__2025-07-10T00:00:00+00:00 [queued]>
[2025-07-25T11:50:20.480+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt_to_dwh.extract_task.load_db_to_parquet scheduled__2025-07-10T00:00:00+00:00 [queued]>
[2025-07-25T11:50:20.480+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-07-25T11:50:20.495+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): extract_task.load_db_to_parquet> on 2025-07-10 00:00:00+00:00
[2025-07-25T11:50:20.503+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'elt_to_dwh', 'extract_task.load_db_to_parquet', 'scheduled__2025-07-10T00:00:00+00:00', '--job-id', '144', '--raw', '--subdir', 'DAGS_FOLDER/elt_to_dwh.py', '--cfg-path', '/tmp/tmpuqfdga0e']
[2025-07-25T11:50:20.506+0000] {standard_task_runner.py:105} INFO - Job 144: Subtask extract_task.load_db_to_parquet
[2025-07-25T11:50:20.507+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=4484) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-25T11:50:20.508+0000] {standard_task_runner.py:72} INFO - Started process 4494 to run task
[2025-07-25T11:50:20.549+0000] {task_command.py:467} INFO - Running <TaskInstance: elt_to_dwh.extract_task.load_db_to_parquet scheduled__2025-07-10T00:00:00+00:00 [running]> on host 74cfddf97016
[2025-07-25T11:50:20.621+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='elt_to_dwh' AIRFLOW_CTX_TASK_ID='extract_task.load_db_to_parquet' AIRFLOW_CTX_EXECUTION_DATE='2025-07-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-10T00:00:00+00:00'
[2025-07-25T11:50:20.622+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-07-25T11:50:21.538+0000] {logging_mixin.py:190} INFO - [Company extract] retrieved 23670 rows from the database
[2025-07-25T11:50:21.571+0000] {logging_mixin.py:190} INFO - Saved data from database to parquet successfully at /opt/airflow/data/parquet/companies/crawl_companies-20250710.parquet
[2025-07-25T11:50:21.572+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-07-25T11:50:21.577+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-07-25T11:50:21.577+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=elt_to_dwh, task_id=extract_task.load_db_to_parquet, run_id=scheduled__2025-07-10T00:00:00+00:00, execution_date=20250710T000000, start_date=20250725T115020, end_date=20250725T115021
[2025-07-25T11:50:21.606+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-25T11:50:21.621+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-25T11:50:21.628+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-07-26T04:56:33.136+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-26T04:56:33.158+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt_to_dwh.extract_task.load_db_to_parquet scheduled__2025-07-10T00:00:00+00:00 [queued]>
[2025-07-26T04:56:33.174+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt_to_dwh.extract_task.load_db_to_parquet scheduled__2025-07-10T00:00:00+00:00 [queued]>
[2025-07-26T04:56:33.175+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-07-26T04:56:33.195+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): extract_task.load_db_to_parquet> on 2025-07-10 00:00:00+00:00
[2025-07-26T04:56:33.203+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'elt_to_dwh', 'extract_task.load_db_to_parquet', 'scheduled__2025-07-10T00:00:00+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/elt_to_dwh.py', '--cfg-path', '/tmp/tmpvi4kzcp0']
[2025-07-26T04:56:33.207+0000] {standard_task_runner.py:105} INFO - Job 20: Subtask extract_task.load_db_to_parquet
[2025-07-26T04:56:33.209+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=569) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-26T04:56:33.210+0000] {standard_task_runner.py:72} INFO - Started process 579 to run task
[2025-07-26T04:56:33.254+0000] {task_command.py:467} INFO - Running <TaskInstance: elt_to_dwh.extract_task.load_db_to_parquet scheduled__2025-07-10T00:00:00+00:00 [running]> on host 3bc71adbf7e0
[2025-07-26T04:56:33.346+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='elt_to_dwh' AIRFLOW_CTX_TASK_ID='extract_task.load_db_to_parquet' AIRFLOW_CTX_EXECUTION_DATE='2025-07-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-10T00:00:00+00:00'
[2025-07-26T04:56:33.347+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-07-26T04:56:35.122+0000] {logging_mixin.py:190} INFO - [Company extract] retrieved 23670 rows from the database
[2025-07-26T04:56:35.249+0000] {logging_mixin.py:190} INFO - Saved data from database to parquet successfully at /opt/airflow/data/parquet/companies/crawl_companies-20250710.parquet
[2025-07-26T04:56:35.250+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-07-26T04:56:35.260+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-07-26T04:56:35.267+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=elt_to_dwh, task_id=extract_task.load_db_to_parquet, run_id=scheduled__2025-07-10T00:00:00+00:00, execution_date=20250710T000000, start_date=20250726T045633, end_date=20250726T045635
[2025-07-26T04:56:35.356+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-26T04:56:35.411+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-26T04:56:35.425+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
